modules/yieldmoBidAdapter.js

```diff
@@ -595,3 +598,14 @@ function shortcutProperty(extraCharacters, target, propertyName) {
 
   return charactersLeft;
 }
+
+/**
+ * Creates and returnes eids arr using createEidsArray from './userId/eids.js' module;
+ * @param {Object} openRtbRequest OpenRTB's request as a cource of userId.
+ * @return array of eids objects
+ */
+function getEids(bidRequest) {
+  if (utils.deepAccess(bidRequest, 'userId')) {
```

Comment:
We have changed how the utils file is imported across prebid to only import the needed functions so you no longer need the utils root. `utils.deepAccess` is now just `deepAccess`

Reply:
resolved